build/scripts/generate_devworkspace_templates.sh

```diff
@@ -18,8 +18,11 @@ do
   if [ -n "$devfile_url" ]; then
     devfile_url=${devfile_url##*v2: }
     devfile_url=${devfile_url%/}
-    devfile_repo=${devfile_url%/tree*}
-    name=$(basename "${devfile_repo}")
+    npm_config_yes=true npx @eclipse-che/che-devworkspace-generator@${CHE_DEVWORKSPACE_GENERATOR_VERSION} \
+        --devfile-url:"${devfile_url}" \
+        --editor-entry:che-incubator/che-code/latest \
+        --output-file:"${dir}"temp.yaml
+    name=$(yq -r '.spec.template.projects[0].name' "${dir}temp.yaml"  | sed -n '2 p')
```

Comment:
probably need to remove temp.yaml after getting name and url from it

Reply:
done